src/Telemetry.ts

```diff
@@ -0,0 +1,30 @@
+import appInsights from "./appInsights";
```

Comment:
Since we're exporting an instance of a class rather than a class, let's change the name of the `Telemetry.ts` file to lowercase to indicate that it doesn't expose just a specific type